products/ocp4/profiles/stig.profile

```diff
@@ -0,0 +1,24 @@
+documentation_complete: true
```

Comment:
We should set this to false until you get the control done. We have done the same for [srg_gpos](https://github.com/ComplianceAsCode/content/blob/master/products/rhel9/profiles/srg_gpos.profile)

Reply:
Addressed